modules/nf-core/bwa/index/main.nf

```diff
@@ -1,6 +1,8 @@
 process BWA_INDEX {
     tag "$fasta"
-    label 'process_single'
+    // NOTE https://bio-bwa.sourceforge.net/bwa.shtml#8
+    // requires 5.37N memory where N is the size of the database
+    memory { 6.B * fasta.size() }
```

Comment:
this is neat! might be a good discussion point in #pipeline-resource-optimization channel and adding in as defaults. Will bring this up in that channel

Reply:
https://github.com/nf-core/modules/pull/6628

Join the on going discussion! 🫠